container_linux_test.go

```diff
@@ -267,3 +272,126 @@ func TestDaemonRestart(t *testing.T) {
 
 	<-statusC
 }
+
+func TestContainerAttach(t *testing.T) {
```

Comment:
There is not much of a difference with directIO for windows because of the pipe and Accept conn